documentation/api.html

```diff
@@ -1458,12 +1458,12 @@
   opacity: 0.5;
 }
 </style>
-    
+
```

Comment:
this file has lots of changes, I guess we need to trust that the tool would have done the right thing? :)

Reply:
Yeah, but then why has this file that hasn't been touched in ages not been fixed already since we run both black and flake8 with every PR?

Reply:
@CodeHeeler good question, I am not sure why this file `api.html` was not caught before.  @fellipeh any thoughts?

Reply:
It cleared out whitespace on empty lines. It should be harmless.